### PR TITLE
Update OCP-29742

### PR DESCRIPTION
**Pull request by weliang1**
Original PR 1079 was closed, open the new one for the same test case.

Fixed two issues suggested by @rbbratta in https://github.com/openshift/verification-tests/pull/1079

Test log:
http://pastebin.test.redhat.com/874651

@zhaozhanqi @huiran0826 @anuragthehatter @rbbratta 
Please help to review, thanks!

### CODE REVIEW

**Comment by zhaozhanqi** [comment]
the steps looks good to me, thanks
/lgtm

File: features/networking/multus.feature [hunk 1]
```diff
@@ -1348,3 +1348,48 @@ Feature: Multus-CNI related scenarios
...
+    
+  # @author weliang@redhat.com
+  # @case_id OCP-29742
+  @admin
+  Scenario: Log pod IP and pod UUID when pod start	
+    Given the multus is enabled on the cluster
+    And I store all worker nodes to the :nodes clipboard
+    # Create the net-attach-def via cluster admin
+    Given I have a project
+    When I run oc create as admin over "<%= BushSlicer::HOME %>/testdata/networking/multus-cni/NetworkAttachmentDefinitions/whereabouts-macvlan.yaml" replacing paths:
```
Comment (pruan-rht) [inline features/networking/multus.feature:1360]: @weliang1 pls use the `Given I obtain...` step to get the testdata.  Check out #1227 for examples

File: features/networking/multus.feature [hunk 2]
```diff
@@ -1348,3 +1348,48 @@ Feature: Multus-CNI related scenarios
...
+      | ["metadata"]["namespace"] | <%= project.name %>      |    
+      | ["spec"]["config"]        |'{ "cniVersion": "0.3.0", "type": "macvlan","mode": "bridge", "ipam": { "type": "whereabouts", "range": "192.168.22.100/24"} }' |
+    Then the step should succeed
+    
+    # Create a pod absorbing above net-attach-def
+    When I run oc create over "<%= BushSlicer::HOME %>/testdata/networking/multus-cni/Pods/generic_multus_pod.yaml" replacing paths:
```
Comment (pruan-rht) [inline features/networking/multus.feature:1366]: @weliang1 pls use the `Given I obtain...` step to get the testdata.  Check out #1227 for examples

File: features/networking/multus.feature (hunk 1 above)
Comment (weliang1) [reply features/networking/multus.feature]: @pruan-rht update to use the way as you suggested. thanks!

File: features/networking/multus.feature (hunk 2 above)
Comment (weliang1) [reply features/networking/multus.feature]: @pruan-rht update to use the way as you suggested. thanks!

**Pull request merged**